helper/tests/helper-tests-allavro/src/test/java/com/linkedin/avroutil1/compatibility/AvroRecordUtilTest.java

```diff
@@ -59,4 +64,57 @@ public void testSupplementDefaultIntoSpecificRecord() throws Exception {
     //should now pass
     AvroCodecUtil.serializeBinary(record);
   }
+
+  @Test
+  public void testLookup() throws Exception {
+
+    ClassLoader cl = Thread.currentThread().getContextClassLoader();
+
+    Class<?> clazz = AvroRecordUtil.lookup(
+            cl,
+            String.class.getName(),
+            null,
+            null,
+            false
+    );
+    Assert.assertSame(clazz, String.class);
+    clazz = AvroRecordUtil.lookup(
+            cl,
+            "DoesNotExist" + UUID.randomUUID(),
+            Collections.singletonList(String.class.getName()),
+            null,
+            false
+    );
+    Assert.assertSame(clazz, String.class);
```

Comment:
Why does this resolve to `String.class`?

Reply:
because its in the aliases